### ISSUE TITLE
Modem not found

### ISSUE THREAD

**Issue by meeximum** [Trixarian/sakis3g-source#9]
Hi,

I'm using the script on a Cubietruck.
I have a problem that lsusb shows the modem (also as modem not as storage), but sakis3g always tells me "Modem not found!"?
Is there a way I can get to root of the problem, any hints welcome :-)

regards
Meex

**Comment by Trixarian** [comment]
It's probably the internal usb_modeswitch list - it's getting pretty old with the last update to it from the end of 2012. I'd recommend installing your distros provided version instead and allowing sakis3g to use that instead. If your device is supported, it should work.

That said, there are some things that won't work with Sakis3G. CDMA and 3G/4G routers are the ones I know about, but newer devices (that has not yet been added to usb_modeswitch) will also not work. There is also a possibility that I messed up the database entries for your device - I had to hack the database to make it work with sakis3g and there is always the possibility that I made a mistake in the 200 odd files I went through.
